src/test/java/gov/nih/nci/evs/api/controller/MdrControllerTests.java

```diff
@@ -260,35 +299,70 @@ public void testMdrSearchContains() throws Exception {
     ConceptResultList list = null;
 
     // Valid test
-    log.info("Testing url - " + url
-        + "?terminology=mdr&fromRecord=0&include=synonyms&pageSize=10&term=Coagulation&type=contains");
-    result = mvc.perform(get(url).header("X-EVSRESTAPI-License-Key", appProperties.getUiLicense())
-        .param("terminology", "mdr").param("term", "Coagulation").param("pageSize", "10").param("type", "contains")
-        .param("include", "synonyms")).andExpect(status().isOk()).andReturn();
+    log.info(
```

Comment:
Another example of very aggressive line breaks compromising readability. It won't affect my approving the PR, but I thought it was worth noting.